ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -661,36 +661,17 @@ Compile Inference Service YAML
     [Arguments]    ${isvc_name}    ${sa_name}    ${model_storage_uri}    ${canaryTrafficPercent}=${EMPTY}
     ...            ${min_replicas}=1   ${scaleTarget}=1   ${scaleMetric}=concurrency  ${auto_scale}=${NONE}
     ...            ${requests_dict}=&{EMPTY}    ${limits_dict}=&{EMPTY}
-    Copy File     ${INFERENCESERVICE_FILEPATH}    ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    ${model_storage_uri}=    Escape String Chars    str=${model_storage_uri}
-    ${rc}    ${out}=    Run And Return Rc And Output
-    ...    sed -i 's/{{INFERENCE_SERVICE_NAME}}/${isvc_name}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    ${rc}    ${out}=    Run And Return Rc And Output
-    ...    sed -i 's/{{MIN_REPLICAS}}/${min_replicas}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    ${rc}    ${out}=    Run And Return Rc And Output
-    ...    sed -i 's/{{SA_NAME}}/${sa_name}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    ${rc}    ${out}=    Run And Return Rc And Output
-    ...    sed -i 's/{{STORAGE_URI}}/${model_storage_uri}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    ${rc}    ${out}=    Run And Return Rc And Output
-    ...    sed -i 's/{{MIN_REPLICAS}}/${min_replicas}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
     IF   '${auto_scale}' == '${NONE}'
-          ${rc}    ${out}=    Run And Return Rc And Output
-          ...    sed -i '/scaleMetric/d' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-          ${rc}    ${out}=    Run And Return Rc And Output
-          ...    sed -i '/scaleTarget/d' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    ELSE
-          ${rc}    ${out}=    Run And Return Rc And Output
-          ...    sed -i 's/{{SCALE_TARGET}}/${scaleTarget}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-          ${rc}    ${out}=    Run And Return Rc And Output
-          ...    sed -i 's/{{SCALE_METRIC}}/${scaleMetric}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    END
-    IF   '${canaryTrafficPercent}' == '${EMPTY}'
-        ${rc}    ${out}=    Run And Return Rc And Output
-        ...    sed -i '/canaryTrafficPercent/d' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
-    ELSE
-        ${rc}    ${out}=    Run And Return Rc And Output
-        ...    sed -i 's/{{CanaryTrafficPercent}}/${canaryTrafficPercent}/g' ${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
+        ${scaleTarget}=    Set Variable    ${EMPTY}
+        ${scaleMetric}=    Set Variable    ${EMPTY}
```

Comment:
could you delete the lines from yaml when autoscaling is none? like done for https://github.com/red-hat-data-services/ods-ci/pull/900/files#diff-343c068d5fc035598a61515b186f393cdae65b3e09fbc9e4326d4bbc6f03da33R675

Reply:
we can check this in a next PR, don't want to block this PR more time.